Cloud-Engineering/L4 - Lead Cloud Engineer.md

```diff
@@ -0,0 +1,78 @@
+# L4: Lead Cloud Engineer
+
+> I am a well-rounded domain expert and leader. I'm trusted to autonomously lead people through complex problems, with a proven record of delivering brand new ideas from inception to production. I deliver high-quality results, and guide technical initiatives using my leadership skills, technical expertise, and a focus on reliability and security.
+
+**Planning horizon:** 6-12 months  
+**Impact radius:** Team (5-15)
+**Evaluation:** Manager or Director
+**Responsibility and direction needed:** Takes ownership of complex projects and leads them from idea to production with limited supervision.
```

Comment:
```suggestion
**Responsibility and direction needed:** Consistently takes ownership of complex projects and leads them from idea to production with limited supervision.
```

Reply:
Introduce "consistently" early on as possible, I do see it's mentioned on specific lines already too.